### DISCUSSION TITLE
Does the same thing exist for the Android version of maplibre?

### DISCUSSION THREAD

**Discussion by ravenfeld** [nazka/mapbox-gl-js-spiderfy#19] [Q&A] [answered]
I find the way of using clusters and rendering them with spiderify very interesting but I use maplibre on Android and I don't see anything close to it.
Do you know of any free software that does the same thing as you do, but in Android?

Thanks in advance

**Comment by simondriesen** [comment, marked as answer]
Sadly, I don’t know of any free or paid software that does the same thing on Android. I also have no experience with the Android version of MapLibre, so porting it to Android would not be easy for me.
